imitation/airl.py

```diff
@@ -31,75 +37,92 @@ def __init__(self, env, gen_policy, discrim, expert_policies, *,
             Due to the way VecEnvs handle
             episode completion states, the last obs-state-obs triple in every
             episode is omitted. (See GitHub issue #1)
-        n_expert_timesteps (int): The number of expert obs-action-obs
+        n_disc_samples_per_buffer (int): The number of
+            obs-state-obs triples sampled from each replay buffer
+            (expert and generator) during
+            each step of discriminator training. This is also the number of
+            triples stored
+            in the replay buffer after each epoch of generator training.
+
+        n_expert_samples (int): The number of expert obs-action-obs
             triples that are generated. If the number of expert policies given
             doesn't divide this number evenly, then the last expert policy
             generates more timesteps.
+        gen_replay_buffer_capacity (Optional[int]): The capacity of the
+            generator replay
+            buffer (the number of obs-action-obs samples from the generator
+            that can be stored).
+
+            By default this is equal to `20 * n_disc_training_samples`.
         init_tensorboard (bool): If True, makes various discriminator
             Tensorboard summaries under the run name "AIRL_{date}_{runnumber}".
             (Generator summaries appear under a different runname because they
             are configured by initializing the stable_baselines policy).
+
+    Raises:
+        ValueError: n_disc_samples_per_buffer > n_expert_samples
```

Comment:
Just removing this bit of documentation for now. Things will become different with __init__ loading expert demonstrations instead of an expert policy later, anyways.